packages/esm-patient-chart-app/src/visit-header/visit-header.component.tsx

```diff
@@ -28,6 +28,7 @@ import { MappedQueuePriority, useVisitQueueEntry } from '../visit/queue-entry/qu
 import { EditQueueEntry } from '../visit/queue-entry/edit-queue-entry.component';
 import VisitHeaderSideMenu from './visit-header-side-menu.component';
 import styles from './visit-header.scss';
+import { useOmrsRestPatient } from '../../../esm-patient-banner-app/src/hooks/usePatientAttributes';
```

Comment:
I don't think this is ideal! Directly importing functionalities from a different app. This delays in terms of performance. You should either create one for that specific app or, for the scoop of this PR, you can use the `usePatient` hook